pyproject.toml

```diff
@@ -161,6 +160,7 @@ ignore = [
   "B007",  # Loop control variable not used within the loop body.
   "B011",  # Don't use assert False
   "B023",  # Function definition does not bind loop variable
+  "B905",  # zip-without-explicit-strict
```

Comment:
This check is quite disruptive IMO and would emit a lot of warnings. I've added it to the ignore list for the time being. https://docs.astral.sh/ruff/rules/zip-without-explicit-strict/